cmd/cmd.go

```diff
@@ -879,6 +883,49 @@ func startMacApp(client *api.Client) error {
 	if err := exec.Command("/usr/bin/open", "-a", path[0]+"Ollama.app").Run(); err != nil {
 		return err
 	}
+	return waitForServer(client)
+}
+
+func executeServer(client *api.Client) error {
```

Comment:
I'm not a fan of this change. It seems like bad form to start a process the user is unaware of without a meaningful of stopping with out `kill`

Reply:
moved this change to #819 for further discussion